index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Andrii Kravchenko">
+    <meta name="description" content="USING HASKELL WITH 5- TO 7-YEAR-OLDS - how to teach children to code on Haskell? Read the article..." >
+    <link rel="icon" href="../myfirstproject/img/Go_gopher_favicon.svg" type="image/x-icon"/>

```

Comment:
Please fix all paths in hrefs so that it corresponds current project folder structure

Reply:
Fix this one please 
![image](https://user-images.githubusercontent.com/104353194/165764182-0fef98c5-2162-4b14-a6f1-d79de509eae2.png)